doc/adr/0016_widget_as_iFrame.md

```diff
@@ -0,0 +1,63 @@
+# 16. Widget as iFrame
+
+Date: 2023-01-10
+
+## Status
+
+Accepted
+
+## Context
+
+We want to provide a standardized entry point for the users in order for them to recognize
+the eID process across different eServices. We believe that a unified user-facing entry point, served as a widget,
+will benefit our customers. To achieve this, there are multiple options. We consider the following three as most feasible:
+
+1. provide framework-specific web components (e.g. react, vue) as a library
```

Comment:
Does the javascript create an iframe here?

Reply:
If you are referring to the component library (react, vue) AFAIK, we would have implemented that without iFrame.